Controller/CkeditorAdminController.php

```diff
@@ -29,16 +30,21 @@ public function browserAction()
     {
         $this->checkIfMediaBundleIsLoaded();
 
-        if (false === $this->admin->isGranted('LIST')) {
-            throw new AccessDeniedException();
+        // NEXT_MAJOR: Remove this when dropping support for SF < 2.8 (change method signature)
```

Comment:
Not sure how to do that on a Controller, I cannot simulate a method signature because SF will not inject me Request if the signature does not contain a parameter with type hinting